cdap-sentry/cdap-sentry-extension/cdap-sentry-binding/src/main/java/co/cask/cdap/security/authorization/sentry/binding/AuthBinding.java

```diff
@@ -638,6 +731,7 @@ protected void revoke(final EntityId entityId, final Role role, Set<Action> acti
   }
 
   private SentryGenericServiceClient getClient() throws Exception {
+    // TODO: Cache the client
```

Comment:
I am not sure I understand this method. Can you just create the client in the constructor and have it as a global variable since I do not see it will change in the code.